app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt

```diff
@@ -11,7 +13,10 @@ import retrofit2.Response
 import java.net.UnknownHostException
 import javax.inject.Inject
 
-class NewsPresenter @Inject constructor(private val mRetrofitServices: RetrofitServices) : BasePresenter<INewsView>() {
+class NewsPresenter @Inject constructor(
+    private val mRetrofitServices: RetrofitServices,
+    private val mSharedPreferencesManager: SharedPreferencesManager
+) : BasePresenter<INewsView>() {
```

Comment:
Try to use identation to read it easily

Reply:
I split the constructor arguments to avoid lines too large breaking code style rules. Which style do you use on these cases?

Reply:
Splitting the constructor is ok. But use some tabs to move line 19 to the right.

Reply:
I tried to fix it, but `ktlint` put it on the left, so I remove the break line.

Reply:
So I put it in one line